packages/local-environment/jest.config.ts

```diff
@@ -19,28 +19,22 @@ export default {
   clearMocks: true,
 
   // Indicates whether the coverage information should be collected while executing the test
-  collectCoverage: true,
+  // collectCoverage: true,
```

Comment:
should be commented? or should be removed?

Reply:
Is there a particular reason why you chose to use the --collectCoverage flag for the coverage command as opposed to using the jest.config.ts? I'm happy its being done consistently, just curious for my own knowledge why you wouldn't always want the coverage vs having a specific coverage command

Reply:
I have absolutely no idea, this is @Imti 
I rebased on his PR and added the contract stuff

Reply:
Noted, disabling collect coverage by default and only having coverage with flag.